src/client/pages/OfferNew/Perils/index.tsx

```diff
@@ -28,17 +30,25 @@ const Wrapper = styled.div`
   }
 `
 
-const PerilRowWrapper = styled.div`
-  padding-bottom: 8rem;
-
-  :last-child {
-    padding-bottom: 0;
-  }
-`
-
 export const Perils: React.FC<Props> = ({ offerData }) => {
   const textKeys = useTextKeys()
 
+  const tabItems = React.useMemo(
+    () =>
+      offerData.quotes.map((quote) => ({
+        id: quote.id,
+        name: quote.displayName,
+        content: <PerilRow offerQuote={quote} />,
+      })),
+    [offerData.quotes],
+  )
+
+  const handleChangeTab = () => {
+    pushToGTMDataLayer({
+      event: 'offer-tab-click',
+    })
```

Comment:
Have we validated this with @sonnyandersson1? Ping @gustaveen or do you already know how this should be implemented?